### DISCUSSION TITLE
Save feeds to boards

### DISCUSSION THREAD

**Discussion by AurelienLeHyaric** [dailydotdev/daily#1186] [Feature Requests]
Hello team,

First, thank you for your wonderful news agreagator tools.

Perhaps this feature has already been requested, I would have liked the possibility of being able to save feeds in boards. Currently we can only bookmark them but it becomes a catch-all that cannot be classified. Having boards by theme would allow us to better classify news to be able to find it more easily. Also another suggestion related to this would be to share the boards and be able to subscribe to other boards in return.

Regards,

Aurélien

**Comment by idoshamun** [comment]
Hi @ALH83,

Do you mean something similar to https://github.com/dailydotdev/daily/discussions/520?

**Reply by AurelienLeHyaric** [reply]
Yes, that's it. thanks

**Reply by idoshamun** [reply]
Closing then, we'll keep the discussion there
